src/plugins/intel_gna/src/backend/gna_limitations.hpp

```diff
@@ -300,14 +301,26 @@ class Limitations {
     constexpr static uint32_t kMemoryPageSize = 4096;
 
 private:
+    struct LimitationsWrapper {
```

Comment:
what is a benefit of introducing new structure for three extra limitations?
Why do they need to be separated.

Reply:
These three are hardware-dependent, in contrast to the above list of limitations which have the same value for all GNA versions